coalib/parsing/CliParsing.py

```diff
@@ -103,4 +103,15 @@ def check_conflicts(sections):
             ArgumentParser().error(
                 "'no_config' cannot be set together 'save' or 'find_config'.")
 
+        try:
+            if (
+                    not section.get('json', False) and
+                    (section.get('output', False) or
+                     section.get('relpath', False))):
+                ArgumentParser().error(
+                    "'output' or 'relpath' cannot be used without `--json`.")
+        except ValueError:
```

Comment:
probably due to a bug? when this isn't present you get this on when you call `--output` you get this error
```python
$ coala --output bleh
[ERROR][01:56:15] An unknown error occurred. This is a bug. We are sorry for the inconvenience. Please contact the developers for assistance. During execution of coala an exception was raised. This should never happen. When asked for, the following information may help investigating:
[INFO][01:56:15] Exception was:
Traceback (most recent call last):

  File "/home/tulphoon/git/coala/coalib/coala_main.py", line 74, in run_coala
    arg_list=arg_list)

  File "/home/tulphoon/git/coala/coalib/settings/ConfigurationGathering.py", line 352, in gather_configuration
    sections, targets = load_configuration(arg_list, log_printer, arg_parser)

  File "/home/tulphoon/git/coala/coalib/settings/ConfigurationGathering.py", line 137, in load_configuration
    check_conflicts(cli_sections)

  File "/home/tulphoon/git/coala/coalib/parsing/CliParsing.py", line 109, in check_conflicts
    (section.get('output', False) or

  File "/usr/lib/python3.5/site-packages/coala_utils/string_processing/StringConverter.py", line 46, in __bool__
    raise ValueError

ValueError
```